GTG/gtk/editor/editor.py

```diff
@@ -60,30 +65,30 @@ def __init__(self,
         self.clipboard = clipboard
         self.builder = Gtk.Builder()
```

Comment:
builder should not be accessed from anywhere else tan this method. Create reference to all buttons that you might need later, connect to all signals. Nothing else should be needed!